File: crates/deslop/tests/fixtures/incremental-multilang/src/ledger_alpha.py
# ledger_alpha.py — the canonical copy of the Python reconciliation routine.

LEDGER_TAG = "ledger"


def reconcile_entries(entries, floor):
    balance = 0
    for entry in entries:
        if entry > floor:
            balance += entry * 2
        else:
            balance -= entry // 2
    return balance

File: crates/deslop/tests/fixtures/incremental-multilang/src/ledger_alpha.go
// ledger_alpha.go — the canonical copy of the Go reconciliation routine.

package ledger

const LedgerTag = "ledger"

func ReconcileEntries(entries []int64, floor int64) int64 {
	var balance int64 = 0
	for _, entry := range entries {
		if entry > floor {
			balance += entry * 2
		} else {
			balance -= entry / 2
		}
	}
	return balance
}
